srcbook/bin/start

```diff
@@ -1,3 +1,7 @@
 #!/usr/bin/env node
 
-import '../index.mjs';
+process.env.NODE_ENV = 'production';
+
+(async () => {
+  await import('../index.mjs');
```

Comment:
Is using `await` here necessary? And if it is, why do we need to wrap it in a function? We use node with top-level await I thought?

Reply:
I'm not sure about using `await`, but the dynamic import is important, otherwise it gets run before setting the process.env.NODE_ENV.

I don't think there's anything wrong with the current syntax?